src/main/java/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,70 @@
+package cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+
+@Entity
+public class ShoppingCart {
+    @Id
+    private Long id;
+    @ManyToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Reply:
we can't have same ticket in two different shopping carts

Reply:
done

Reply:
changed